src/components/Tabs/Tabs.stories.tsx

```diff
@@ -78,7 +78,7 @@ Playground.argTypes = {
 Playground.args = { selectedValue: 'inventory' };
 
 export const OnlyTabs: Story = () => (
-  <Tabs selectedValue="overview" onSelectTab={action('Select Tab')}>
+  <Tabs selectedValue="inventory" onSelectTab={action('Select Tab')}>
```

Comment:
@dclerici-ssc why did you changed this? In this case should be active the second tab with the custom color as a example of the customization